icu4c/source/tools/toolutil/writesrc.cpp

```diff
@@ -19,7 +19,7 @@
 */
 
 #include <stdio.h>
-#include <inttypes.h>
+#include <cinttypes>
 #include <time.h>
```

Comment:
optional, to put the C++ standard header after the C standard headers:
```suggestion
#include <time.h>
#include <cinttypes>
```

Reply:
@markusicu Done.